projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="lg-side-nav-bar-item__container">
+  <div class="lg-side-nav-bar-item__title lg-font-size-1--strong">{{ title }}</div>
```

Comment:
Initially I thought this should be a header for a11y reasons, but I notice that isn't the case for the current tabs, so maybe not. @elenagarrone?

Reply:
Yes, I would definitely have it as a heading

Reply:
And i would also add an `aria-describedby` to the description connecting it to the title

Reply:
Ignore my comments above as I'm happy with what we've talked about on the call.